video/out/vo_gpu_next.c

```diff
@@ -1269,6 +1269,10 @@ static bool draw_frame(struct vo *vo, struct vo_frame *frame)
         case PL_COLOR_TRC_BT_1886:
         case PL_COLOR_TRC_GAMMA22:
         case PL_COLOR_TRC_SRGB:
+#ifdef __APPLE__
```

Comment:
Color management is also always active on Wayland if the compositor gives us a preferred image description, as well as on Windows with ACM/HDR. This should do the same for all 3 platforms